packages/api/cms-api/src/auth/strategies/static-authed-user.strategy.ts

```diff
@@ -1,27 +1,32 @@
-import { Injectable } from "@nestjs/common";
+import { Inject, Injectable, Optional } from "@nestjs/common";
 import { PassportStrategy, Type } from "@nestjs/passport";
-import jwt from "jsonwebtoken";
-import { ExtractJwt, Strategy } from "passport-jwt";
+import { Strategy } from "passport-custom";
 
 import { CurrentUserInterface } from "../current-user/current-user";
+import { CURRENT_USER_LOADER, CurrentUserLoaderInterface } from "../current-user/current-user-loader";
 
 interface StaticAuthedUserStrategyConfig {
-    staticAuthedUser: CurrentUserInterface;
+    staticAuthedUser?: CurrentUserInterface;
+    staticAuthedUserId?: string;
```

Comment:
We could also achieve this with `staticAuthedUser: CurrentUserInterface | string` and a typeof check. Would remove the edge case where neither option is given.